### PR TITLE
T/207: getChangedFilesForCommit() sometimes does not work as it should

### LINKED ISSUE #207: getChangedFilesForCommit() sometimes does not work as it should
Described here https://github.com/ckeditor/ckeditor5-dev/issues/184#issuecomment-301878542 and https://github.com/ckeditor/ckeditor5-dev/issues/184#issuecomment-301899131.

Possible solution: https://github.com/ckeditor/ckeditor5-dev/pull/173#issuecomment-297648588

### PR DESCRIPTION
**Pull request by pomek**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Fix: A changed way for gathering changed files in a commit. Closes #207.

### CODE REVIEW

**Comment by Reinmar** [comment]
CI failed.

**Comment by pomek** [comment]
Should be good now.

File: packages/ckeditor5-dev-env/lib/release-tools/utils/transform-commit/getchangedfilesforcommit.js [hunk 1]
```diff
@@ -14,12 +14,20 @@ const { tools } = require( '@ckeditor/ckeditor5-dev-utils' );
  * @returns {Array.<String>}
  */
 module.exports = function getChangedFilesForCommit( commitId ) {
-	const gitCommand = `git diff --name-only ${ commitId }~..${ commitId }`;
+	const gitCommand = `git log -m -1 --name-only --pretty="format:" ${ commitId }`;
 	const changedFiles = tools.shExec( gitCommand, { verbosity: 'error' } ).trim();
 
-	if ( !changedFiles ) {
-		return [];
```
Comment (Reinmar) [inline packages/ckeditor5-dev-env/lib/release-tools/utils/transform-commit/getchangedfilesforcommit.js]: You can make this simpler:

image

You don't need to check whether `\n\n` is in the string.

Comment (pomek) [inline packages/ckeditor5-dev-env/lib/release-tools/utils/transform-commit/getchangedfilesforcommit.js]: Will fix it. Thx.

Comment (pomek) [inline packages/ckeditor5-dev-env/lib/release-tools/utils/transform-commit/getchangedfilesforcommit.js]: @Reinmar, done.

**Comment by Reinmar** [comment]
I'm not sure if it's gonna work – let's see in practice ;)

**Comment by pomek** [comment]
I hope… :D

**Pull request merged**